### PR TITLE
M5

### PR DESCRIPTION
**Pull request by rgozim**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -29,3 +20,12 @@ dependencies {
...
 }
+
+gradlePlugin {
+    plugins {
+        BlitzPlugin {
+            id = "org.openmicroscopy.blitz"
+            implementationClass = "org.openmicroscopy.blitz.BlitzPlugin"
+        }
+    }
+}
```
Comment (jburel) [inline build.gradle]: add extra line

File: src/main/groovy/org/openmicroscopy/blitz/ImportHelper.groovy [hunk 2]
```diff
@@ -2,38 +2,42 @@ package org.openmicroscopy.blitz
...
 import org.gradle.api.artifacts.Configuration
 import org.gradle.api.artifacts.DependencySet
-import org.gradle.api.artifacts.ResolvedArtifact
+import org.gradle.api.artifacts.ResolvableDependencies
+import org.gradle.api.logging.Logger
+import org.gradle.api.logging.Logging
+import org.gradle.api.plugins.JavaPlugin
 
-import static org.openmicroscopy.blitz.ConventionPluginHelper.getRuntimeClasspathConfiguration
+import static org.openmicroscopy.blitz.ConventionPluginHelper.*
```
Comment (jburel) [inline src/main/groovy/org/openmicroscopy/blitz/ImportHelper.groovy]: explicit import

File: src/main/groovy/org/openmicroscopy/blitz/tasks/ImportResourcesTask.groovy [hunk 3]
```diff
@@ -37,6 +37,7 @@ class ImportResourcesTask extends DefaultTask {
     @TaskAction
     void apply() {
         ResolvedArtifact artifact = config.resolvedConfiguration.resolvedArtifacts.find {
+            Log.info("Resolved: ${it.file.toString()}")
```
Comment (jburel) [inline src/main/groovy/org/openmicroscopy/blitz/tasks/ImportResourcesTask.groovy]: Do you want to keep it?

File: build.gradle [hunk 4]
```diff
@@ -1,20 +1,11 @@
 plugins {
     id "groovy"
-    id "org.openmicroscopy.plugin-project" version "5.5.0-m4"
+    id "org.openmicroscopy.plugin-project" version "5.5.0-SNAPSHOT"
```
Comment (jburel) [inline build.gradle]: SNAPSHOT should not be used. Travis will always fail

File: src/main/groovy/org/openmicroscopy/blitz/ImportHelper.groovy [hunk 5]
```diff
@@ -2,38 +2,42 @@ package org.openmicroscopy.blitz
...
-        if (!config) {
-            config = createDataFilesConfig(project)
+    static File findOmeroModel(ResolvableDependencies dependencies) {
+        File file = dependencies.artifacts.artifactFiles.files.find {
+            it.name.contains("omero-model")
+        }
+        if (file == null) {
+            throw new GradleException("Unable to find omero-model artifact")
         }
+        Log.info("Found omero-model: ${file.toString()}")
```
Comment (jburel) [inline src/main/groovy/org/openmicroscopy/blitz/ImportHelper.groovy:54]: To be kept?

**Comment by jburel** [comment]
This PR is on top of #8

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by jburel** [comment]
tested locally as part of the m5 prep

**Pull request merged**